src/pages/txs/index.tsx

```diff
@@ -66,9 +64,24 @@ export default function Transactions({
           {isLoading ? (
             <SkeletonLoader rows={7} screen={SkeletonLoaderScreen.Tx} />
           ) : (
-            data.transactions.map((tx) => (
-              <TransactionRow key={tx.hash} rawData={tx} />
-            ))
+            data.transactions.map((tx) => {
+              const transformedData = transformTransactionData(tx);
+              return (
+                <TransactionRow
+                  key={tx.hash}
+                  data={{
+                    transactionType: transformedData.transactionType,
+                    from: transformedData.from,
+                    to: transformedData.to,
+                    hash: transformedData.hash,
+                    amount: transformedData.amount,
+                    symbol: transformedData.symbol,
+                    timeInSec: transformedData.timeInSec,
+                    status: transformedData.status,
+                  }}
+                />
+              );
+            })
```

Comment:
```suggestion
            data.transactions.map((tx) => (
                <TransactionRow
                  key={tx.hash}
                  data={transformTransactionData(tx)}
                />
              )
            )
```

Reply:
same as above, `data`'s type is not the same as what `transformTransactionData` returns

Reply:
actually you're right, it works this way. Thank you! 🎉